modules/Liftover.py

```diff
@@ -0,0 +1,117 @@
+import gcsfs
+
+from pyliftover import LiftOver
+
+import pyspark.sql.types as T
+import pyspark.sql.functions as F
```

Comment:
If Pyspark functions are imported as F, the instances need to be updated.

I have a preference for using an upper case as it is the way I most commonly see them referenced. It is a pain when you want to reproduce some snippet in the REPL and I have an error because of differences in how the API is imported.
Databricks import them using upper case in [their docs](https://docs.gcp.databricks.com/spark/latest/dataframes-datasets/introduction-to-dataframes-python.html), also from the[ Data Analysis with Python and Pyspark book](https://books.google.pt/books?id=4ytfEAAAQBAJ&pg=PA53&lpg=PA53&dq=from+pyspark.sql+import+functions+as+F+or+as+f&source=bl&ots=24AtuuBozV&sig=ACfU3U2K-N-de_fBhZXRlAB2hPqL1NpjMw&hl=es&sa=X&ved=2ahUKEwi83OKxuoL5AhXvm_0HHaE5C1YQ6AF6BAg4EAM#v=onepage&q=from%20pyspark.sql%20import%20functions%20as%20F%20or%20as%20f&f=false)

Again, no hard feelings, I just prefer to make a single decision and adapt the rest of the code to that.